src/tests/functional/plugin/gpu/single_layer_tests/dynamic/pad.cpp

```diff
@@ -129,40 +224,29 @@ INSTANTIATE_TEST_SUITE_P(
                 ::testing::ValuesIn(padsBegin2D_Smoke),
                 ::testing::ValuesIn(padsEnd2D_Smoke),
                 ::testing::Values(0),
+                ::testing::ValuesIn(isConstantInput),
                 ::testing::ValuesIn(padMode)),
         PadLayerGPUTest::getTestCaseName
 );
 
-/* *======================* *=====================* *======================* */
-
-/* *======================* Dynamic Shapes Tests 4D *======================* */
-
+//====================== Dynamic Shapes Tests 4D ======================
 const std::vector<InputShape> inputShapesDynamic4D = {
-        {{-1, -1, -1, -1},                                    // dynamic
-         {{5, 36, 5, 5}, {3, 16, 10, 5}, {3, 24, 10, 10}}},   // target
-
-        {{-1, 32, -1, -1},                                    // dynamic
-         {{5, 32, 5, 5}, {5, 32, 5, 8}, {3, 32, 8, 8}}},      // target
-
-        {{{1, 5}, {16, 32}, {1, 16}, {1, 16}},                // dynamic
-         {{3, 16, 5, 5}, {5, 24, 5, 8}, {3, 32, 8, 8}}},      // target
+    {{-1, -1, -1, -1}, {{5, 36, 5, 5}, {3, 16, 10, 5}}}
```

Comment:
Could you add shape < 4 ?

Reply:
There are 2d shape tests, 'smoke_GPUPadDynamic2DConst' and 'smoke_GPUPadDynamic2D'.